chaos_genius/views/anomaly_data_view.py

```diff
@@ -425,6 +357,46 @@ def kpi_anomaly_retraining(kpi_id):
         return jsonify({"msg": f"retraining failed for KPI: {kpi_id}, KPI id is None"})
 
 
+def _get_dimensions_values(kpi_id):
+    """Creates a dictionaryy of KPI dimension and their values.
+
+    :param kpi_id: ID of the KPI
+    :type row: integer
+    :return dimensions_vals: dictionary of {dimension:list(vals)}
+    :rtype: dictionary
+    """
+    # Get unique list of subdims and values from DB
+    query = (
+        db.session.query(
+            func.distinct(AnomalyDataOutput.series_type)
+        )
+        .filter(
+            (AnomalyDataOutput.kpi_id == kpi_id)
+            & (AnomalyDataOutput.anomaly_type == "subdim")
+        )
```

Comment:
What do you think about adding a time bound here? Maybe get only the last 60 or 90 days of data, so that the performance wouldn't degrade too much over time.

Reply:
It's a good idea, but there are some caveats to it:
- We might miss some subdims that appeared before the 90 or 60 days mark. Although that won't matter because we only display the anomaly time period data. 
I'll think about this.

Reply:
Yes. Ideally, it would be good to check only for data after the anomaly time period. If that needs additional data or queries, even harcoding 90 days will let it scale better over time.

Reply:
Done.